_posts/2016-06-20-alpha-release-notes.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title:  "argparse2cwl alpha release notes"
+date:   2016-06-20 02:27:27 +0300
+comments: true
+---
+
+`argparse2cwl` forms CWL tool descriptions from programs which use `argparse` as their argument parser. 
+
+## Prerequisites ##
+`argparse2cwl` is compatible with both Python 2 & 3.
+Dependencies:
+* Jinja2 >= 2.8
+
+## Installing ##
+
+Installation process:
+
+1. Clone the source repository:
```

Comment:
Wait a second, it works out of the box with the newest `cwl-refactor` changes:

```
Finished processing dependencies for gxargparse==0.2.5
$ gxargparse_check_path
Ready to go!
```

Also, let's have `master` as a "release" branch, so that we can skip step 2 (checkout out `cwl-refactor` branch).

Getting there, good job Anton! ;)